src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    public static void greet() {
+        System.out.println("_________________________________________");
```

Comment:
Perhaps the underscores can be assigned as a private static variable such that you do not have to manually enter the them in the code multiple times.